helm-deployment/management-api-subchart/templates/management.yaml

```diff
@@ -106,4 +107,9 @@ spec:
   selector:
     app: management-api
   type: ClusterIP
-
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: mgt-api
```

Comment:
maybe add to name information that this is account? or use a convention - "management-api"